tests/api.c

```diff
@@ -14731,6 +14736,143 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
 
+/* Testing wc_SignatureGetSize() */
+static int test_wc_SignatureGetSize(void) {
+
+    int ret = 0; 
+    enum wc_SignatureType sig_type;
+    word32 key_len;
+    ecc_key ecc; 
```

Comment:
ECC-specific API that comes from <wolfssl/wolfcrypt/ecc.h> should be protected with an "#ifdef HAVE_ECC".  This will compile out those sections when the user has disabled ECC.

Also, since the test vector being used below is on the SECP256R1 curve, the ECC code here should be compiled out if the user has disabled ECC256 support.  In total this would look something like:

```
#if defined(HAVE_ECC) && !defined(NO_ECC256)
...
#endif
```